src/explorer/CosmosDBTreeItem.ts

```diff
@@ -119,6 +132,16 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
                 await createdDatabase.cosmosExtensionItem.reveal();
             }
         });
+        if (createdDatabase.cosmosExtensionItem.azureData && createdDatabase.cosmosExtensionItem.connectionString.startsWith('postgres://')) {
```

Comment:
I don't like having multiple prompts at once. How about you add a "Manage Firewall Rules" button to the above prompt?

As an aside - you see how adding this new code (aka the firewall stuff) caused Nathan and I to add more comments to the PR? This is actually one pitfall to avoid that can keep PRs open for longer. It's best to focus on existing comments so that a longstanding PR can get merged, and work on new enhancements separately after that's done